java/c/src/test/python/integration_tests.py

```diff
@@ -399,6 +399,9 @@ def recreate_batch():
                     return reader.read_next_batch()
 
         self.round_trip_record_batch(recreate_batch)
+    
+    def test_runendencoded_array(self):
+        self.round_trip_array(lambda: pa.RunEndEncodedArray.from_arrays([3, 5, 10, 12, 19], [1, 2, 1, None, 3]),  check_metadata=False)
```

Comment:
```suggestion
        self.round_trip_array(lambda: pa.RunEndEncodedArray.from_arrays([3, 5, 10, 12, 19], [1, 2, 1, None, 3]), check_metadata=False)
```

Reply:
Can we also test empty arrays, and ideally one other type here (strings?)

Reply:
sure